experiments/functional/cspc-pool/block-deletion-with-volume/run_litmus_test.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: verify-cspc-pool-deletion-if-pool-with-volume-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: verify-cspc-pool-deletion-
```

Comment:
- should be removed in label value. And change the label value too according to the job name